enterprise/api_client/lms.py

```diff
@@ -238,7 +238,7 @@ def get_course_enrollment(self, username, course_id):
         # This enrollment data endpoint returns an empty string if the username and course_id is valid, but there's
         # no matching enrollment found
         if not result:
-            LOGGER.error('no course enrollment details found for username=%s, course=%s', username, course_id)
+            LOGGER.info('no course enrollment details found for username=%s, course=%s', username, course_id)
```

Comment:
@mattdrayer There's one at https://github.com/edx/edx-enterprise/pull/220/files#diff-c06e09cde492a184aeafc6e12c32fe49R232 right above this too -- would you want that to be `info`?

Reply:
@UmanShahzad The error log message you pointed out above should stay as error since that is an unexpected scenario.

Nit: While you are in here perhaps you could improve the message a bit:
```
LOGGER.info('Failed to find course enrollment details for user [%s] and course [%s]', username, course_id)
```